content/posts/2012/12/12/snoonet-updates.md

```diff
@@ -1,6 +1,6 @@
 ---
 date: 2012-12-12T00:00:00
-draft: false
+draft: true
 title: "Snoonet Updates"
 type: "post"
 ---
```

Comment:
We no longer support support [at] snoonet.org.

Out of date SSL fingerprint information.